packages/stress/src/utils/stressClient.ts

```
@@ -186,12 +194,23 @@ export class StressClient {
         return channelId
     }
 
-    async startStreamsClient(metadata: { spaceId: string }) {
+    async startStreamsClient(config: Parameters<typeof this.streamsClient.initializeUser>[0]) {
         if (isDefined(this.streamsClient.userStreamId)) {
             return
         }
-        await this.streamsClient.initializeUser(metadata)
+        let device: ExportedDevice | undefined
+        try {
+            const rawDevice = await fs.readFile(this.deviceFilePath, 'utf8')
+            device = JSON.parse(rawDevice) as ExportedDevice
+        } catch {
+            return
```

You ignored the error here! :)

---

thank you!! 🙏